### ISSUE TITLE
TextEdit::GetTextBounds regression in v0.13.2 

### ISSUE THREAD

**Issue by wahern** [yue/yue#161]
Yue Version: 0.13.2
Operating System: macOS

The TextEdit::GetTextBounds fix seems to have had the complete opposite effect as intended. I have code which automatically adjusts the minheight of a TextEdit widget on size change so all the lines of wrapped text would be displayed. This worked correctly up to 0.13.1. With 0.13.2 :gettextbounds returns a height for two lines even when the text clearly isn't wrapped; the returned text bounds height doesn't shrink (from 2 lines to 1) until there's a considerable amount of whitespace at the end of the text.

**Comment by wahern** [comment]
Note: The issue occurs from 0.13.2 all the way to the latest release, 0.13.6.

**Comment by zcbenz** [comment]
I confirm the bug, thanks for reporting!

code snippet for testing:

```lua
local gui = require('yue.gui')

local win = gui.Window.create{}
win.onclose = function() gui.MessageLoop.quit() end

local edit = gui.TextEdit.create()
edit:setstyle{flex=1}
win:setcontentview(edit)

edit:settext('1')
local minheight = edit:gettextbounds().height

function edit.ontextchange()
  local height = edit:gettextbounds().height
  if height < minheight then
    height = minheight
  end
  local size = win:getcontentsize()
  size.height = height
  win:setcontentsize(size)
end

win:setcontentsize{width=200, height=minheight}
win:center()
win:activate()

gui.MessageLoop.run()
```

**Comment by zcbenz** [comment]
Closed with e53947c5de2baf8fb517525d38d63e581091b001.
